src/app/tasks/check-inbox-dialog/client-details-dialog/client-details-dialog.component.html

```diff
@@ -0,0 +1,102 @@
+<h2 mat-dialog-title>{{ 'labels.inputs.Client Details' | translate}}</h2>
+
+<mat-dialog-content class="dialog-content">
+
+  <div *ngIf="loading" class="spinner-container">
+    <mat-progress-spinner mode="indeterminate"></mat-progress-spinner>
+  </div>
+
+  <div *ngIf="!loading && client" class="content-container">
+    <div class="images-row">
+      <div class="image-block">
+        <span>{{ 'labels.commons.Client Image' | translate }}</span>
```

Comment:
remove spaces in transaction codes
```suggestion
        <span>{{ 'labels.commons.client.image' | translate }}</span>
```
Idem for ID image, etc